### ISSUE TITLE
Updated IPA for IOS

### ISSUE THREAD

**Issue by Mara-Li** [vendetta-mod/Vendetta#231]
Hello.

Sorry to bother you directly on the repository, but I can't get any response to this issue in the Discord (as i already made some discussion in the forum about it).

It's since october we didn't get any update of Vendetta for IOS in the IPA thread. 
Comparatively, Enmity and Rosicord have already launched the last updated discord (213 for Rosi, more updated than Enmity).

What cause this issue ? Do you drop the support for IOS ?

In many way, I largely prefer to use Vendetta over Enmity, but I don't want to use an old version. 

For information, I tried to update the IPA myself, but I'm stuck with the sign step and It seems that the methods I used wasn't good, as i have a lot of issue of my patched version.
I have set as release my IPA [here](https://github.com/Lisandra-dev/VendettaTweak/releases/tag/v213.0) and linked some fix for the GURU MEDITATION error. 

If possible, can we (ie User of Vendetta on IOS) an update about the issue ?

Thanks.

**Comment by maisymoe** [comment]
Vendetta still supports iOS - it's just an unfortunate situation in which our iPod we used for IPA extraction quite literally blew up - and my own replacement device (second-hand from ebay, with a bad battery) can be temperamental.

Combine that with the fact that IPA releases are a two-person process, in which *I* must extract the IPA, send it to Fiery, and then *he* patches and uploads it, and I think the problem becomes clearer.

I apologise for the inconvenience.

**Comment by Dreel0akl** [comment]
How the fuck did your iPod blow up

**Comment by maisymoe** [comment]
> How the fuck did your iPod blow up

battery become pillow

**Comment by acquitelol** [comment]
> Combine that with the fact that IPA releases are a two-person process, in which *I* must extract the IPA, send it to Fiery, and then *he* patches and uploads it, and I think the problem becomes clearer.

Consider making the patcher open source, that way anyone can just patch their own IPAs (including you) and then you can upload the IPA via a webhook for simplicity

**Comment by scarlet1nbythehill** [comment]
i am pretty sure the patcher is open source

**Comment by OneTapTrap** [comment]
so yall just want my developer license to just potentially get this added to the app store or at the bear minimum not have to refresh it every week lol.  all my signed apps are good for a year

**Comment by madeline-yana** [comment]
I'm sorry for barging in, are they *specifically* asking *you* for your developer license? Because like, there's TrollStore, and there's certificate fuckery, as alternatives... But TrollStore is limited to 14.0-16.6 and 17.0, and certificates have always been fucky. 

tl;dr im confused

**Comment by Opabina** [comment]
is there an ETA on when iOS will receive updates again?
